ignite-base/App/Themes/Colors.js

```diff
@@ -1,7 +1,9 @@
 // @flow
 
-const colors = {
-  background: '#1F0808',
+import { merge } from 'ramda'
+
+const igniteColors = {
+  igniteBackground: '#1F0808',
```

Comment:
@jamonholmgren no, it is not namespaced at all. Colors.background is currently set to the values set in Colors, which will not match what a user wants for their own ignited app. The goal here is to decouple Ignite's default settings for colors - in particular Colors.background, from the Color.background that a user might want for their own Ignited app. So when you first generate a project via Ignite, the pre-ignited screens (which I love and want to preserve!) use the background color generated for all pre-ignited screens. But an Ignited app should be able to use Colors.background for their own app without breaking the presentation of pre-ignited screens such as ignite/ignite-base/App/Containers/Styles/ListviewExampleStyle.js